docs/source/techniques/ISIS_SingleCrystalDiffraction_Workflow.rst

```diff
@@ -176,7 +176,7 @@ The reduction proceeds as follows:
         sxd.save_peak_table(run, peak_type, integration_type, save_dir, save_format='SHELX')
 
     # save combined table
-    sxd.save_all_peaks(peak_type, integration_type, save_dir=save_dir, save_format=fmt)
+    sxd.save_all_peaks(peak_type, integration_type, save_dir=save_dir, save_format='Jana')
```

Comment:
Sorry I know you asked which format to use but I didn't have internet at the time! But now I can see it, this should actually be `SHELX` to be consistent with L176!